Sources/TensorFlow/Core/Tensor.swift

```diff
@@ -40,12 +40,12 @@ public struct Tensor<Scalar: TensorFlowScalar> {
   @usableFromInline
   internal var _isScalarZero = false
 
-  /// An internal workaround for SR-13263: debug info generation crash.
-  @usableFromInline
-  class SR13263Workaround {}
+  // /// An internal workaround for SR-13263: debug info generation crash.
+  // @usableFromInline
+  // class SR13263Workaround {}
 
-  /// An internal workaround for SR-13263: debug info generation crash.
-  internal var _sr13263Workaround: SR13263Workaround?
+  // /// An internal workaround for SR-13263: debug info generation crash.
+  // internal var _sr13263Workaround: SR13263Workaround?
```

Comment:
The underlying issue has migrated in that you can now trigger the same crash by inserting this placeholder, rather than avoiding the crash by having it. I've seen this with other types. Unclear when the behavior inverted.

Reply:
I'm not seeing this happen with or without `SR13263Workaround` commented out, on current Swift toolchains.

Reply:
I'm not seeing the crash affect S4TF, with or without `SR13623Workaround` commented out. I'm running `swift build` and `swift test`, and it's not activating the crash on recent toolchains. Am I doing this right?

```bash
export TENSORFLOW_USE_RELEASE_TOOLCHAIN=1
cd s4tf
swift build -Xswiftc -DTENSORFLOW_USE_STANDARD_TOOLCHAIN \
  -c release -Xswiftc -g -Xcc -I${DESTDIR}/usr/include -Xlinker -L${DESTDIR}/usr/lib

rm -f .build/release/libx10.dylib
cp ${DESTDIR}/usr/lib/libx10.dylib .build/release/libx10.dylib

swift test -Xswiftc -DTENSORFLOW_USE_STANDARD_TOOLCHAIN \
    -c release -Xswiftc -g -Xcc -I${DESTDIR}/usr/include -Xlinker -L${DESTDIR}/usr/lib
```